code/index.html

```diff
@@ -1,22 +1,228 @@
+      }
+    </script>
+
+    <h1>NEWS TIME</h1>
+
+    </div>
+    </div>
+  </header>
+
+  <!--THE NAV MENU UNDER THE HEADER FROM 668px and up-->
+  <nav class="menu">
+    <a href="#">News</a>
+    <a href="#">World</a>
+    <a href="#">Local</a>
+    <a href="#">Cultur</a>
+    <a href="#¢">Politics</a>
+    <a href="#">Sport</a>
+  </nav>
+
+
+
+
+  <section class="cards">
+
+    <div class="article-one">
+        <div class="card">
+          <img src="pexels-pixabay-73821.jpg" alt="Hexbot image">
+          <div class="fadedbox">
+            <div class="title text"><button>Article</button> 
+            </div>
+          </div>
+        </div>
+    </div>
+    </a>
+
+    <div class="article-two">
+        <div class="card">
+          
+          <h4>Big Article</h4>
+          <h5>Nearly 50 dead after Afghan downpour.At least 46 people have been killed, about 80 have been injured in the city of Charikar north of
+            Afghanistan's capital Kabul. At least 46 people have been killed, about 80 have been injured in the.
+              </h5>
+          <p>At least 46 people have been killed, about 80 have been injured in the city of
+            Charikar north of
+            Afghanistan's capital Kabul.At least 46 people have been killed, about 80 have been injured in the city of Charikar north of
+            Afghanistan's capital Kabul. At least 46 people have been killed, about 80 have been injured in the city of
+            Charikar north of
+            Afghanistan's capital Kabul. At least 46 people have been killed, about 80 have been injured in the city of
+            Charikar north of
+            Afghanistan's capital Kabul. At least 46 people have been killed, about 80 have been injured in the city of
+            Charikar north of
+            Afghanistan's capital Kabul. At least 46 people have been killed, about 80 have been injured in the city of
+            Charikar north of
+            Afghanistan's capital Kabul. At least 46 people have been killed, about 80 have been injured in the city of Charikar north of
+            Afghanistan's capital Kabul. At least 46 people have been killed, about 80 have been injured in the city of
+            Charikar north of
+            Afghanistan's capital Kabul. At least 46 people have been killed, about 80 have been injured in the city of
+            Charikar north of
+            Afghanistan's capital Kabul. At least 46 people have been killed, about 80 have been injured in the city of
+            Charikar north of
+            Afghanistan's capital Kabul. At least 46 people have been killed, about 80 have been injured in the city of
+            Charikar north of
+           </p>
+          <div class="fadedbox">
+            <div class="title text"><button>Article</button> </div>
+          </div>
+        </div>
+      </div>
+    </a>
+
+
+    <!--card 1-->
+    <div class="card">
+        <h4>NEWS</h4>
+        <img src="people-3111875_640.jpg" alt="Hexbot image">
```

Comment:
Nice use of alt in the image tag 🏆 For upcoming projects it might also be a good idea to store your pictures in a separate image folder to make it easier and less messy for yourself!

Reply:
when I tried that all the images from the page disappears, so how do I put them in a folder correctly?